app/views/users/edit.html.erb

```diff
@@ -1,23 +1,13 @@
-<div class="row">
-  <div class="col m2">
-
-  </div>
-  <div class="col m8 account-create-card">
-    <div class="card grey darken-1 large">
-      <div class="card-content white-text">
-        <span class="card-title">Create An Account</span>
-        <%= form_for(current_user) do |f| %>
-        <a><%= f.text_field :name, placeholder: 'Name' %></a>
-
-        <a><%= f.text_field :email, placeholder: 'Email' %></a>
-
-        <a><%= f.text_field :address, placeholder: 'Address' %></a>
-
-        <a><%= f.password_field :password, placeholder: 'Password' %></a>
-
-        <a class="waves-effect waves-light btn teal darken-4"><%= f.submit("Update") %></a>
-        <% end %>
-      </div>
-    </div>
+<div class="card grey darken-1 large">
+  <div class="card-content white-text">
+    <span class="card-title">Edit Your Account <small>(All Fields must be completed to update profile, including password field)</small></span>
+    <hr>
+    <%= form_for(current_user) do |f| %>
+    <p>Name: <%= f.text_field :name, placeholder: 'Name' %></p>
+    <p>Email: <%= f.text_field :email, placeholder: 'e-mail' %></p>
```

Comment:
I just made a small change to change 'Email' to 'e-mail'.  This seems like a better way to do it.  Let me know if you disagree.